data/gravity-model/prem.txt

```diff
@@ -0,0 +1,50 @@
+# A gravity profile that follows the description of
+# the preliminary reference Earth model (PREM) by
+# Dziewonski & Anderson (1981)
```

Comment:
Can you add a doi here? 
All Earth scientists will know what you mean anyway, but @bangerth will complain about it ;-)